### ISSUE TITLE
Consider option for tracking idle factories on core selector

### ISSUE THREAD

**Issue by sprunk** [ZeroK-RTS/Zero-K#5121]
Commanders and nanos can be tracked, why not factories?
![image](https://github.com/ZeroK-RTS/Zero-K/assets/2573076/95237745-c46e-4d6e-9e83-c93cb196ea85)
![image](https://github.com/ZeroK-RTS/Zero-K/assets/2573076/11c05d2b-7d9f-407e-9a2b-3db97ccd0aa8)

**Comment by GoogleFrog** [comment]
As in with the Z hotkey? Because core selection currently tracks idle factories by showing a picture of the factory, rather than a unit with a progress bar on it.

**Comment by sprunk** [comment]
Yes, with the Z hotkey. The picture is nice but checking it is an extra step when I mash Z every now and then anyway. A possible caveat is that facs sometimes do get obsolete and idle so it should be possible to mark them as busy (eg via wait).
